ui/src/settings/segmenters/details/config/SegmentersActivitySection.js

```diff
@@ -23,6 +23,7 @@ export const SegmentersActivitySection = ({ segmenter }) => {
         textStyle="reverse"
         type="responsiveColumn"
         listItems={items}
+        columnWidths={["110px", "auto"]}
```

Comment:
Although this file didn't have a corresponding scss file, I'm using the same dimensions found in https://github.com/caraml-dev/xp/pull/79/files#diff-dcd7230faa44847d72337909b45f268964bfa0bea37a2203fe28ea9e9cd08c9eR31 (since the components look extremely similar).